Library/Homebrew/os/mac.rb

```diff
@@ -47,19 +47,15 @@ def cat
     end
 
     def languages
-      return @languages unless @languages.nil?
+      return @languages if defined?(@languages)
```

Comment:
The `@languages ||= begin` pattern I think is generally nicest here but don't feel super strongly.